pom.xml

```diff
@@ -631,6 +634,14 @@ limitations under the License.
           </execution>
         </executions>
       </plugin>
+      <plugin>
+        <groupId>org.sonatype.central</groupId>
+        <artifactId>central-publishing-maven-plugin</artifactId>
+        <extensions>true</extensions>
+        <configuration>
+          <publishingServerId>central</publishingServerId>
+        </configuration>
+      </plugin>
```

Comment:
I don't think this needs to be added to both the pluginManagement section and the regular plugins section, does it? I think the pluginManagement section just needs to specify the version. The extension and config should only go here, not in the management section.

Reply:
Most likely correct.  Typically, I keep all my config in dep management, so I only followed my pattern.  But yes, the extension bit needs held in plugin run.  Config may not be necessary there, haven't tried that.

Reply:
I cherry-picked your change here, but omitted the configuration, since it is the default value, and can be overridden with a property.